torchvision/models/googlenet.py

```diff
@@ -62,8 +69,9 @@ def __init__(self, num_classes=1000, aux_logits=True, transform_input=False, ini
         self.inception5a = Inception(832, 256, 160, 320, 32, 128, 128)
         self.inception5b = Inception(832, 384, 192, 384, 48, 128, 128)
 
-        self.aux1 = InceptionAux(512, num_classes)
-        self.aux2 = InceptionAux(528, num_classes)
+        if aux_logits:
```

Comment:
This partially reverts https://github.com/pytorch/vision/pull/821 from @TheCodez 
I've made a comment in https://github.com/pytorch/vision/pull/821#discussion_r270351338 which is related to this change, and I think that @TheCodez reasoning was good.

Reply:
the reasoning makes sense. so should we do something similar in the inception model?